.travis/linux.win32.install.sh

```diff
@@ -1,16 +1,30 @@
 #!/usr/bin/env bash
+set -e
 
-PACKAGES="nsis cloog-isl libmpc3 qt4-linguist-tools mingw32
-	mingw32-x-sdl mingw32-x-libvorbis mingw32-x-fluidsynth mingw32-x-stk
+if [ $QT5 ]; then
+	MINGW_PACKAGES="mingw32-x-qt5base"
+else
+	MINGW_PACKAGES="mingw32-x-qt"
+fi
+
+MINGW_PACKAGES="mingw32-x-sdl mingw32-x-libvorbis mingw32-x-fluidsynth mingw32-x-stk
 	mingw32-x-glib2 mingw32-x-portaudio mingw32-x-libsndfile mingw32-x-fftw
 	mingw32-x-flac mingw32-x-fltk mingw32-x-libsamplerate
 	mingw32-x-pkgconfig mingw32-x-binutils mingw32-x-gcc mingw32-x-runtime
-	mingw32-x-libgig mingw32-x-libsoundio"
+	mingw32-x-libgig mingw32-x-libsoundio $MINGW_PACKAGES"
 
-if [ $QT5 ]; then
-	PACKAGES="$PACKAGES mingw32-x-qt5base"
-else
-	PACKAGES="$PACKAGES mingw32-x-qt"
-fi
+export MINGW_PACKAGES
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
@tresf introduced this line in 67334a84c270e38a8d5a209970db073445e40bad, I copied it. Maybe he can explain the motivation behind it.